<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>h264Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_controller" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>511</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>18</horstretch>
      <verstretch>29</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_controller
{
	background-color:rgb(27,100,100);
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QWidget" name="widget_video" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_open">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{ 
image: url(:image/open_normal.png);
border-radius:0px; 
} 
QPushButton:hover{ 
image: url(:image/open_focus.png);
border-radius:0px; 
} 
QPushButton:pressed{ 
image: url(:image/open_normal.png);
border-radius:0px; 
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_play">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{ 
image: url(:image/start_normal.png);
border-radius:0px; 
}  
QPushButton:hover{ 
image: url(:image/start_focus.png);
border-radius:0px; 
} 
QPushButton:pressed{ 
image: url(:image/start_normal.png);
border-radius:0px; 
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_pause">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{ 
image: url(:image/pause_normal.png);
border-radius:0px; 
} 
QPushButton:hover{ 
image: url(:image/pause_focus.png);
border-radius:0px; 
} 
QPushButton:pressed{ 
image: url(:image/pause_normal.png);
border-radius:0px; 
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_stop">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{ 
image: url(:image/stop_normal.png);
border-radius:0px; 
} 
QPushButton:hover{ 
image: url(:image/stop_focus.png);
border-radius:0px; 
} 
QPushButton:pressed{ 
image: url(:image/stop_normal.png);
border-radius:0px; 
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_container" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_container
{
	background-color: rgb(50,50, 86);
	border:1px solid rgba(0, 0, 0, 30);
	border-radius:2px;  /*边框拐角*/
}
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_show_video"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:15px;color:blue</string>
    </property>
    <property name="text">
     <string>欢迎使用mapping远程播放器 http://blog.cvosrobot.com/</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:15px;color:black</string>
    </property>
    <property name="text">
     <string>当前状态：</string>
    </property>
   </widget>
   <widget class="QLabel" name="constatus">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>未连接</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>461</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(184,184,147);</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;IP地址:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditaddr">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.31.135</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;端口：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditport">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>5556</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtoncnt">
     <property name="geometry">
      <rect>
       <x>224</x>
       <y>40</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>62</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>拍照</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>开始录制</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxrate">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>300x300</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1280x720</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>分辨率：</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关于</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>461</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(184,184,147);</string>
    </property>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>后退</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stop_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_run">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>前进</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ri">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>右转</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_le">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>左转</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>98</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>显示机器人</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>显示轨迹</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>显示规划路径</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_automapping">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>自主建图</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_manualmapping">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>手动建图</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_loadmapping">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>170</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>加载地图</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_setgoal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>设定目标点并规划路径</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cleargoal">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>220</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清除目标和路径</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_explore">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>自主探索</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_switch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>启动</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_savmap">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>保存地图</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_a">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>140</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_v">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.05</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>线速度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>140</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>角速度：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mapnavigation">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>220</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>地图自主导航</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
